<launch>
    <!--Uncomment line below to run simulation -->
    <!--<include file="$(find rbe3002)/launch/practice_maze.launch" />-->

	<include file="$(find rbe3002)/launch/rviz_mapping.launch" />

    <!--<include file="$(find rbe3002)/launch/gazebo_rviz_mapping.launch" />-->
    <node name="expand_map" pkg="rbe3002" type="Expand_Map.py" output="screen"/>

    <node name="frontier_finder" pkg="rbe3002" type="Frontier_Finder.py" output="screen"/>

    <node name="a_star" pkg="rbe3002" type="A_Star.py" output="screen"/>

    <node name="robot_nav" pkg="rbe3002" type="Robot.py" output="screen"/>

    <!--<node name="has_been_described_as_the_Beyonce_of_nodes" pkg="rbe3002" type="Controller.py" />-->


</launch>
